core/library_steam.gd

```diff
@@ -363,7 +369,8 @@ func _app_info_to_launch_item(info: Dictionary, is_installed: bool) -> LibraryLa
 	item.provider_app_id = app_id
 	item.name = data["name"]
 	item.command = "steam"
-	item.args = ["-gamepadui", "-steamos3", "-steampal", "-steamdeck", "-silent", "steam://rungameid/" + app_id]
```

Comment:
I think this change creates issues with **Steam**. I have no focus on the game, but on a black window and there is no controls. When I use the official stable release the games I tested work. MH Rise Demo and Pixel Gun 3D. Also I noticed that the UI is in BigPicture mode somehow.

Reply:
reverting that change make the controls work again. Also the correct window is focused.